dom.bs

```diff
@@ -1028,6 +1028,18 @@ specified otherwise it returns null.
 <p class="note no-backref"><a>Nodes</a>, <a for=/>shadow roots</a>, and <a>documents</a> override
 the <a>get the parent</a> algorithm.
 
+<p>Each {{EventTarget}} object can have an associated
+<dfn export for=EventTarget>legacy pre-click-dispatch steps</dfn> algorithm,
+<dfn export for=EventTarget>legacy post-canceled-click-dispatch steps</dfn> algorithm, and
+<dfn export for=EventTarget>activation behavior</dfn> algorithm. The <a>activation behavior</a>
+algorithm is passed an <a>event</a>, as indicated in the <a>dispatch</a> algorithm.</p>
+
+<p class="note no-backref">An {{EventTarget}} object that has <a>legacy pre-click-dispatch steps</a>
```

Comment:
This phrasing in the non-normative note seems a little strange. I think it's stating an observation about the world, but maybe it's meant to be more of a requirement?

Perhaps a better conceptual framework would be to define two special types of EventTargets:

- clickable event targets, which have activation behavior
- clickable event targets with interception, which additionally have legacy pre-click dispatch steps and legacy post-canceled-click-dispatch steps

(better names welcome). Then you say e.g. "every HTMLXElement is a clickable event target with the following activation behavior: ..."